templates/index.html

```diff
@@ -1497,6 +1497,14 @@ <h2>
       secondaryUrl.href = selectedTakeover.secondary_url;
       secondaryUrl.innerHTML = selectedTakeover.secondary_cta + "&nbsp;&rsaquo;";
     }
+
+    dataLayer.push({
+      event: "NonInteractiveGAEvent",
+      eventCategory: "www.ubuntu.com-impression-takeover",
```

Comment:
This is `cn` not `www.` but maybe better categorised as just a "Chinese takeover impression"